modules/block-interactions/src/main/java/unnamed/mmo/blocks/handlers/FarmlandHandler.java

```diff
@@ -0,0 +1,112 @@
+package unnamed.mmo.blocks.handlers;
+
+import net.kyori.adventure.sound.Sound;
+import net.minestom.server.coordinate.Point;
+import net.minestom.server.entity.Entity;
+import net.minestom.server.entity.EntityType;
+import net.minestom.server.entity.Player;
+import net.minestom.server.entity.metadata.item.ItemEntityMeta;
+import net.minestom.server.instance.Instance;
+import net.minestom.server.instance.block.Block;
+import net.minestom.server.instance.block.BlockHandler;
+import net.minestom.server.item.ItemStack;
+import net.minestom.server.item.Material;
+import net.minestom.server.sound.SoundEvent;
+import net.minestom.server.utils.NamespaceID;
+import org.jetbrains.annotations.NotNull;
+import unnamed.mmo.blocks.BlockInteractionUtils;
+
+import java.util.Map;
+
+public class FarmlandHandler implements BlockHandler {
+
+    @Override
+    public void onDestroy(@NotNull Destroy destroy) {
+        Point cropPosition = destroy.getBlockPosition().add(0, 1, 0);
+        Instance instance = destroy.getInstance();
+        if(cropMap.containsKey(instance.getBlock(cropPosition).registry().material())) {
+            // Drop item
+            Entity entity = new Entity(EntityType.ITEM);
+            CropBlockData data = cropMap.get(instance.getBlock(cropPosition).registry().material());
+            if(entity.getEntityMeta() instanceof ItemEntityMeta itemEntityMeta) {
+                itemEntityMeta.setItem(ItemStack.of(data.cropBlockMaterial()));
+            }
+            entity.setInstance(instance, cropPosition);
+            // Break it too
+            instance.setBlock(cropPosition, Block.AIR);
+            // Minestom please better particle methods
+            // TODO Particle effects
+            // Probably want to PR this into Minestom, but a instance.playSound(Sound, Point) convenience method
```

Comment:
Can add to utility, or pr to minestom, or use the fork in the org

Reply:
I'll probably want to add a ParticleUtils to the project, and PR the `instance.playSound(Sound, Point);` method to the repo itself.